presto-main/src/test/java/com/facebook/presto/type/TestRowOperators.java

```diff
@@ -156,17 +156,31 @@ public void testIsDistinctFrom()
     }
 
     @Test
-    public void testRowEquality()
+    public void testRowComparison()
             throws Exception
     {
         assertFunction("row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2001-01-02 03:04:05.321 +07:10') = " +
```

Comment:
I believe these tests are intended for the same instant in different timezones. Please fix these and your test.

You can put this change in the same commit as the one that removes `try` usage in this class, or a different one.

e.g. `2001-01-02 03:04:05.321 +07:09 = 2001-01-02 03:04:06.321 +07:10`